src/plugins/telemetry/public/services/telemetry_sender.ts

```diff
@@ -53,13 +53,22 @@ export class TelemetrySender {
     return false;
   };
 
+  private delayRetrySend = () => {
+    return 60 * (1000 * Math.min(Math.pow(2, this.retryCount), 64)); // 120s, 240s, 480s, 960s, 1920s, 3840s, 3840s, 3840s
+  };
+
   private sendIfDue = async (): Promise<void> => {
-    if (this.isSending || !this.shouldSendReport()) {
```

Comment:
Given the removal of `isSending`, could we add unit tests where `sendIfDue` is called multiple time in a row without delay just to be sure that `shouldSendReport` properly handles it (aka `sendUsageData` should only be called once)

Reply:
`sendIfDue` is private and we’re the only ones calling the [method](https://github.com/elastic/kibana/pull/117955/files#diff-5a45ad5f1ca947b4b167609152746b1599cf51aac704cd7b181bb73cec63e208R104). It will never be called without a delay.